### PR TITLE
Pr81x  FatEvent Filters to use event count from TCDS FED 1024

### PR DESCRIPTION
**Pull request by rekovic**
Fixes to fat event filters, to use 64-bit L1 event count from TCDS FED 1024, instead of 24-bit L1 event number from DAQ header.  (from PR https://github.com/cms-l1t-offline/cmssw/pull/367)

### CODE REVIEW

**Comment by perrotta** [comment]
please test

**Comment by Martin-Grunewald** [comment]
+1

File: HLTrigger/special/src/HLTL1NumberFilter.cc [hunk 1]
```diff
@@ -56,6 +58,7 @@ HLTL1NumberFilter::fillDescriptions(edm::ConfigurationDescriptions& descriptions
   desc.add<unsigned int>("period",4096);
   desc.add<bool>("invert",true);
   desc.add<int>("fedId",812);
```
Comment (fwyzard) [inline HLTrigger/special/src/HLTL1NumberFilter.cc]: can you change the `fedId` to 1024 by default ?

File: HLTrigger/special/src/HLTL1NumberFilter.cc [hunk 2]
```diff
@@ -37,7 +38,8 @@ HLTL1NumberFilter::HLTL1NumberFilter(const edm::ParameterSet& config) :
   inputToken_( consumes<FEDRawDataCollection>(config.getParameter<edm::InputTag>("rawInput")) ),
   period_( config.getParameter<unsigned int>("period") ),
   fedId_(  config.getParameter<int>("fedId") ),
-  invert_( config.getParameter<bool>("invert") )
+  invert_( config.getParameter<bool>("invert") ),
+  useFED1024L1Count_( config.getParameter<bool>("useFED1024L1Count") )
```
Comment (fwyzard) [inline HLTrigger/special/src/HLTL1NumberFilter.cc]: if the new mode only works with the TCDS FED, could you add a check at construction time that `useFED1024L1Count` is set only if `fedId == 1024` ?

**Comment by fwyzard** [comment]
@rekovic , can you check what happens when using the new mode over some recent data ?

**Comment by Martin-Grunewald** [comment]
-1
OK, so more work needed....

**Comment by davidlange6** [comment]
@rekovic - any plans to update this PR?

**Pull request closed without merging**